packages/vue-components/src/Modal.vue

```diff
@@ -109,16 +114,39 @@ export default {
       return this.small ? 'modal-sm' : 'modal-lg';
     },
     optionalCentering() {
-      return this.center ? 'modal-dialog-centered' : '';
+      return this.center && this.scrollBehavior === 'inside' ? 'modal-dialog-centered' : '';
     },
     effectClass() {
       return this.effect === 'zoom' ? this.zoomEffect : 'vfm';
     },
+    modalContentClass() {
+      return this.scrollBehavior === 'outside'
+        ? 'modal-content-scroll-outside' : 'modal-content-scroll-inside';
+    },
   },
   methods: {
     close() {
       this.show = false;
     },
+    opened() {
+      // To adjust modal styles according to scrollBehavior
+
+      const modal = $vfm.get(this.id)[0].$el;
```

Comment:
Do the properties mentioned [here](https://v2.vue-final-modal.org/guide/properties/) help you in the stuff you are doing below?